diff -up desmume-master/desmume/src/texcache.cpp.2~ desmume-master/desmume/src/texcache.cpp
--- desmume-master/desmume/src/texcache.cpp.2~	2021-10-27 18:51:35.765537647 +0200
+++ desmume-master/desmume/src/texcache.cpp	2021-10-27 18:52:50.437630424 +0200
@@ -1166,10 +1166,10 @@ void NDSTextureUnpack4x4(const size_t sr
 	for (size_t y = 0, d = 0; y < yTmpSize; y++)
 	{
 		size_t tmpPos[4] = {
-			((y<<2)+0) * sizeX,
-			((y<<2)+1) * sizeX,
-			((y<<2)+2) * sizeX,
-			((y<<2)+3) * sizeX
+			static_cast<size_t>(((y<<2)+0) * sizeX),
+			static_cast<size_t>(((y<<2)+1) * sizeX),
+			static_cast<size_t>(((y<<2)+2) * sizeX),
+			static_cast<size_t>(((y<<2)+3) * sizeX)
 		};
 		
 		for (size_t x = 0; x < xTmpSize; x++, d++)
